kimchi/src/verifier.rs

```diff
@@ -166,7 +166,8 @@ where
             //~~ - Derive the scalar joint combiner challenge $j$ from $j'$ using the endomorphism.
             //~~   (TODO: specify endomorphism)
             let joint_combiner = ScalarChallenge(joint_combiner);
-            let joint_combiner = (joint_combiner, joint_combiner.to_field(&index.srs().endo_r));
```

Comment:
joint_combiner is implicitly copied at pos 0 before when `ScalarChallenge` derives `Copy`

Reply:
And now it transfers the ownership instead, no copies occur